Makefile

```diff
@@ -1216,7 +1216,7 @@ clean-rocks:
 	$(FIND) . -name "*.[oda]" -exec rm -f {} \;
 	$(FIND) . -type f -regex ".*\.\(\(gcda\)\|\(gcno\)\)" -exec rm -f {} \;
 
-clean-rocksjava:
+clean-rocksjava: clean-rocks
```

Comment:
Here is the error message, it return error code 2 (find and delete failed because dir jl is just deleted by `clean-rocksjava`:
```
$ make rocksdbjava && make clean
...
find . -name "*.[oda]" -exec rm -f {} \;
cd java && /Library/Developer/CommandLineTools/usr/bin/make clean
find: ./jl: No such file or directory
rm -rf ./include
rm -rf test-libs
rm -rf target
make: *** [clean-rocks] Error 1
make: *** Waiting for unfinished jobs....
...
```